pgml-dashboard/src/lib.rs

```diff
@@ -519,3 +522,176 @@ pub fn routes() -> Vec<Route> {
 pub async fn migrate(pool: &PgPool) -> anyhow::Result<()> {
     Ok(sqlx::migrate!("./migrations").run(pool).await?)
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::components::sections::footers::MarketingFooter;
+    use crate::guards::Cluster;
+    use crate::utils::config;
+    use rocket::fairing::AdHoc;
+    use rocket::http::{Cookie, Status};
+    use rocket::local::asynchronous::Client;
+
+    #[sqlx::test]
+    async fn test_remove_modal() {
+        let rocket = rocket::build().mount("/", routes());
+        let client = Client::untracked(rocket).await.unwrap();
+
+        let cookie = vec![
+            NotificationCookie {
+                id: "1".to_string(),
+                time_viewed: Some(chrono::Utc::now() - chrono::Duration::days(1)),
+                time_modal_viewed: Some(chrono::Utc::now() - chrono::Duration::days(1)),
+            },
+            NotificationCookie {
+                id: "2".to_string(),
+                time_viewed: None,
+                time_modal_viewed: None,
+            },
+        ];
+
+        let response = client
+            .get("/notifications/product/modal/remove_modal?id=1")
```

Comment:
This feels like the wrong place to implement this. This should be in our web app, and we should pass it through the context into the dashboard.

Reply:
Not sure what you mean here.  The test, or the endpoint should be in web app? I felt like both are needed here since notifications include marketing notifications.